airgun/entities/host.py

```diff
@@ -153,6 +153,20 @@ def schedule_remote_job(self, entities_list, values, timeout=60, wait_for_result
             status_view.wait_for_result(timeout=timeout)
         return status_view.read()
 
+    def play_ansible_roles(self, entities_list, timeout=60, wait_for_results=True):
```

Comment:
LGTM ..
having a query around **timeout=60** , Is it enough time play role without specifying a custom timeout ?

Reply:
This is enough for the majority of small jobs (as the job is for immediate running), but yes if the test intend to use some huge roles  or if a host contain many roles to apply it can customize that timeout value.